src/db.rs

```diff
@@ -400,7 +435,7 @@ impl RecordDatabase {
             .query_map([], |row| {
                 if let ValueRef::Text(bytes) = row.get_ref_unwrap(0) {
                     // SAFETY: the underlying data is always valid utf-8
-                    f(unsafe { std::str::from_utf8_unchecked(bytes) });
+                    f(std::str::from_utf8(bytes).unwrap());
```

Comment:
So this adds a check before the conversion. Why is it used in favour of the unsafe block?

Reply:
It was for consistency with some other conversions; in principle I think if the underlying database is malformed we should panic rather than cause undefined behaviour. Almost this will not result in any meaningful performance impact (I imagine it takes quite a bit longer to read the data from the SQLite database into memory but this should be benchmarked I suppose)

Reply:
I see! I was just curious. In general I think I'd prefer `unwrap` as well.